core/test/financial_templates/Liquidatable.js

```diff
@@ -256,61 +292,63 @@ contract("Liquidatable", function(accounts) {
       });
-      it("Liquidation has already been disputed", async () => {
-        await liquidationContract.dispute(liquidationParams.uuid, sponsor, { from: disputer });
-        // Mint enough tokens to disputer for another dispute bond
-        await collateralToken.mint(disputer, disputeBond, { from: contractDeployer });
-        await collateralToken.increaseAllowance(liquidationContract.address, disputeBond, { from: disputer });
-        assert(
-          await didContractThrow(liquidationContract.dispute(liquidationParams.uuid, sponsor, { from: disputer }))
-        );
-        assert.equal((await collateralToken.balanceOf(disputer)).toString(), disputeBond.toString());
-      });
-      // Weird edge cases, test anyways:
-      it("Liquidation already disputed successfully", async () => {
+      it("Dispute initiates an oracle call", async () => {
+        const disputeTime = await liquidationContract.getCurrentTime();
         await liquidationContract.dispute(liquidationParams.uuid, sponsor, { from: disputer });
-        await liquidationContract.settleDispute(
-          liquidationParams.uuid,
-          sponsor,
-          { rawValue: settlementPrice.toString() },
-          true
-        );
-        // Mint enough tokens to disputer for another dispute bond
-        await collateralToken.mint(disputer, disputeBond, { from: contractDeployer });
-        assert(
-          await didContractThrow(liquidationContract.dispute(liquidationParams.uuid, sponsor, { from: disputer }))
+        // Oracle should have an enqueued price after calling dispute
+        const pendingRequests = await mockOracle.getPendingQueries();
+        assert.equal(
+          pendingRequests[0]["identifier"],
+          priceTrackingIdentifier.padEnd(pendingRequests[0]["identifier"].length, 0)
         );
-        assert.equal((await collateralToken.balanceOf(disputer)).toString(), disputeBond.toString());
+        assert.equal(pendingRequests[0]["time"], disputeTime);
       });
-      it("Liquidation already disputed unsuccessfully", async () => {
+      it("Throw if liquidation has already been disputed", async () => {
         await liquidationContract.dispute(liquidationParams.uuid, sponsor, { from: disputer });
-        await liquidationContract.settleDispute(
-          liquidationParams.uuid,
-          sponsor,
-          { rawValue: settlementPrice.toString() },
-          false
-        );
         // Mint enough tokens to disputer for another dispute bond
         await collateralToken.mint(disputer, disputeBond, { from: contractDeployer });
+        await collateralToken.increaseAllowance(liquidationContract.address, disputeBond, { from: disputer });
         assert(
           await didContractThrow(liquidationContract.dispute(liquidationParams.uuid, sponsor, { from: disputer }))
         );
         assert.equal((await collateralToken.balanceOf(disputer)).toString(), disputeBond.toString());
       });
+      // Weird edge cases, test anyways:
```

Comment:
Why not delete these?

Reply:
I actually think some of these should be included 🤔 . Will look a bit deeper and update the PR